ASTree.cpp

```diff
@@ -282,6 +282,23 @@ PycRef<ASTNode> BuildFromCode(PycRef<PycCode> code, PycModule* mod)
                 stack.push(new ASTMap());
             }
             break;
+        case Pyc::BUILD_CONST_KEY_MAP_A:
+            // Top of stack will be a tuple of keys.
+            // Values will start at TOS - 1.
+            {
+                PycRef<ASTNode> keys = stack.top();
+                stack.pop();
+
+                ASTConstMap::values_t values;
```

Comment:
Could put a `values.reserve(operand)` to pre-allocate the vector...

Reply:
Done!